docs/standard/library-guidelines/sourcelink.md

```diff
@@ -0,0 +1,24 @@
+---
+title: SourceLink
+description: Best practice recommendations for using SourceLink to improve debugging for .NET libraries.
+author: jamesnk
+ms.author: jamesnk
+ms.date: 09/20/2018
+---
+# SourceLink
+
+SourceLink is a technology that enables source code debugging of .NET assemblies from NuGet by developers. When creating a NuGet package you use the SourceLink tooling to embed source control metadata inside assemblies and the package. Developers who download the package and have SourceLink enabled in Visual Studio can step into the source code using the source control metadata to create a great debugging experience.
+
+[![Newtonsoft.Json SourceLink Demo](./media/sourcelink-youtube-thumbnail.png)](https://www.youtube.com/watch?time_continue=61&v=gyRGhCQPkB4)
+
+## Using SourceLink
+
+Instructions for using SourceLink can be found on the [SourceLink GitHub repository](https://github.com/dotnet/sourcelink/blob/master/README.md).
+
+You can use [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer) to confirm SourceLink metadata has been successfully embedded in the package:
```

Comment:
One thing to note: NPE is not actually showing SourceLink metadata, it's showing the ["Repository" metadata from the NuSpec](https://docs.microsoft.com/en-us/nuget/reference/nuspec#repository). The SourceLink package happens to add both but the presence of this metadata doesn't necessarily guarantee that SourceLinked PDBs are present.

I'm not sure exactly how (or even if) to express that in the guide though. It's probably OK to just leave this content as-is, since as long as you're using the SourceLink package it will work.

Reply:
How about this:

> You can use [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer) to confirm SourceLink metadata has been successfully embedded in the package. Make sure the `Repository` metadata is present and .pdb files are located next to all .dll.